frontend/packages/dev-console/src/components/helm/form/HelmInstallUpgradeForm.tsx

```diff
@@ -140,7 +142,7 @@ const HelmInstallUpgradeForm: React.FC<FormikProps<FormikValues> & HelmInstallUp
         isSubmitting={status?.isSubmitting || isSubmitting}
         submitLabel={helmAction}
```

Comment:
We need to translate the `submitLabel` as well. `helmAction` is an enum which we could not translate directly, so we need a mapping here from enum to label, or?

Reply:
We can no translate this enum as it also use to show the form type of `Install`, `Upgrade` or `Rollback`.

Reply:
What's about 

```suggestion
        submitLabel={resetLabel={t(`devconsole~${helmAction}`)}}
```

?

Not perfect but should work. Don't know if we have some other areas where we need to translate enums. But if I'm right the good thing is that if we add an enum the translation shows as label (helmaction) as fallback. In our case we need to add all three labels also if they doesn't existing 1:1 in sourcecode. (I mean, there is no reverse lookup like t(`devconsole~Install`))

Reply:
Same in `HelmReleaseRollbackForm.tsx`